qemu/tests/enable_scatter_windows.py

```diff
@@ -172,28 +176,55 @@ def _get_driver_version(session):
         lambda: _is_process_finished(session, autoit_name), timeout, 20, 3
     )
 
-    error_context.context("Check if wireshark is installed", test.log.info)
-    check_installed_cmd = params.get("check_installed_cmd")
-    check_result = session.cmd_output(check_installed_cmd)
-    if "tshark" not in check_result:
-        error_context.context("Install wireshark", test.log.info)
-        install_wireshark_cmd = params.get("install_wireshark_cmd")
-        install_wireshark_cmd = utils_misc.set_winutils_letter(
-            session, install_wireshark_cmd
-        )
-        status, output = session.cmd_status_output(
-            install_wireshark_cmd, timeout=timeout
-        )
-        if status:
-            test.error(
-                "Failed to install wireshark, status=%s, output=%s" % (status, output)
+    tools = {
+        "windump": {
+            "check_cmd": params.get("check_windump_installed_cmd"),
+            "install_cmd": utils_misc.set_winutils_letter(
+                session, params.get("windump_install_cmd")
+            ),
+            "process_name": params.get("windump_name"),
+        },
+        "wireshark": {
+            "check_cmd": params.get("check_wireshark_installed_cmd"),
+            "install_cmd": utils_misc.set_winutils_letter(
+                session, params.get("wireshark_installation_cmd")
+            ),
+            "process_name": "tshark",
+        },
+    }
```

Comment:
This simply does not make sense.

```
for tool in ['windump', 'tshark']:
    check_cmd = params.get("check_%s_installed_cmd" % tool)
    install_cmd = utils_misc.set_winutils_letter(session, params.get("%s_install_cmd" % tool))
```
And replace params `wireshark_*` to `tshark_*` in cfg.

Reply:
Also it should be params[""] instead of params.get(), as if check_cmd is idle the following steps would fail.